.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,17 @@
+<!--
+
+ PLEASE MAKE SURE NO SENSIBLE INFORMATION ABOUT YOUR
+ SYSTEM IS EXPOSED WHEN SUBMITTING FEEDBACK!
+
+ Thank you for considering giving feedback to the project!
+
+ When adding your feedback/bug please keep in mind:
+   - This is not a support channel.
+     Please go to https://scc.suse.com/support for support.
+   - When posting bugs/feedback here, there is no guarantee that the issue
+     will be handled immediately. Depending on the topic and the availability
+     of the developers, the issue maybe resolved in undefined time or
+     even ignored.
```

Comment:
I believe we are not going to ignore any issue, there is a chance we are going to react and close something we know is impossible at the moment. Let's, maybe, remove `or even ignored.` part ?

Reply:
fair enough. just wanted to make clear there is also the chance it will not work out. But yes right might sound a little bit rude.